README.md

```diff
@@ -109,6 +109,10 @@ Guide to running Nydus with fscache: [Nydus-fscache](./docs/nydus-fscache.md)
 
 Nydus is deeply integrated with [Dragonfly](https://d7y.io/) P2P system, which can greatly reduce the network latency and the single point of network pressure for registry server, testing in the production environment shows that using Dragonfly can reduce network latency by more than 80%, to understand the performance test data and how to configure Nydus to use Dragonfly, please refer to the [doc](https://d7y.io/docs/setup/integration/nydus).
 
+## Accelerate OCI v1 image directly with Nydus
```

Comment:
please also refine
`- Native [eStargz](https://github.com/containerd/stargz-snapshotter) image support with remote snapshotter plugin `nydus-snapshotter` for containerd runtime.` here

Reply:
Still have some issues with estargz support in the master branch, will fix them on next PR.

Reply:
## Accelerate OCI image directly with Nydus

Let's don't introduce some version since there is no OCI v2.

Reply:
Fixed